<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TTRecordWindow</class>
 <widget class="QMainWindow" name="TTRecordWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QFrame" name="frame">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>1</number>
        </property>
        <item>
         <widget class="QWebView" name="webView">
          <property name="url">
           <url>
            <string>about:blank</string>
           </url>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="2">
         <widget class="QComboBox" name="titleBox">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="3">
         <widget class="QPlainTextEdit" name="noteTextEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="overwriteMode">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="addNoteButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Добавить</string>
          </property>
          <property name="shortcut">
           <string>Alt+D</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;Новая нота</string>
          </property>
          <property name="buddy">
           <cstring>titleBox</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Запрос</string>
    </property>
    <addaction name="actionEditRecord"/>
    <addaction name="actionSaveChanges"/>
    <addaction name="actionCancelChanges"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Значение</string>
    </property>
    <addaction name="actionSaveExit"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRefresh"/>
   <addaction name="separator"/>
   <addaction name="actionEditRecord"/>
   <addaction name="actionSaveChanges"/>
   <addaction name="actionCancelChanges"/>
  </widget>
  <action name="actionEditRecord">
   <property name="icon">
    <iconset resource="tracks.qrc">
     <normaloff>:/images/edit.png</normaloff>:/images/edit.png</iconset>
   </property>
   <property name="text">
    <string>Редактировать</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionSaveChanges">
   <property name="icon">
    <iconset resource="tracks.qrc">
     <normaloff>:/images/ok.png</normaloff>:/images/ok.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить изменения</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCancelChanges">
   <property name="icon">
    <iconset resource="tracks.qrc">
     <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
   </property>
   <property name="text">
    <string>Отменить изменения</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="tracks.qrc">
     <normaloff>:/images/view-refresh.png</normaloff>:/images/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>Обновить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Закрыть</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionSaveExit">
   <property name="text">
    <string>Сохранить и закрыть</string>
   </property>
   <property name="toolTip">
    <string>Сохранить изменения и закрыть окно</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Return</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="tracks.qrc"/>
 </resources>
 <connections/>
</ui>
